BP/history/provinces/bp_k_italia.txt

```diff
@@ -413,105 +475,120 @@
 }
 2645 = {	#CALATAFIMI
 	holding = none
-	#Void to leave empty holdings
 }
 2644 = {	#MAZARA
-	holding = none
+	holding = church_holding #For diversity
 }
-
 ###c_messina
 2633 = {	#MESSINA
 	culture = greek
 	religion = hellenic_pagan
 	holding = castle_holding
+	buildings = {
+		common_tradeport_02
+	}
+	584.1.1 = { religion = nicene }
 }
 2634 = {	#CEFALU
 	holding = city_holding
 }
 2639 = {	#CATANIA
-	holding = church_holding
+	holding = church_holding #for diversity
 }
-
 ###c_siracusa
 2638 = {	#SYRACUSE
 	culture = greek
-	religion = hellenic_pagan
+	religion = nicene #Early center of Christianity
 	holding = castle_holding
+	buildings = {
+		curtain_walls_00
+		common_tradeport_02
+		cereal_fields_02
+	}
 }
 2643 = {	#LENTINI
-	holding = none
+	holding = city_holding
+	847.1.1 = { holding = none } #destroyed by Saracens
 }
 2642 = {	#CALTAGIRONE
-	holding = city_holding
+	holding = none
 }
-
 ###c_agrigento
 2637 = {	#AGRICENTO
 	culture = greek
 	religion = hellenic_pagan
 	holding = castle_holding
+	buildings = {
+		common_tradeport_00
+	}
+	529.1.1 = { religion = nicene }
 }
 2640 = {	#CASTROGIOVANNI
-	holding = none
+	holding = city_holding
 }
 2641 = {	#CALTANISETTA
-	holding = city_holding
+	holding = church_holding #for diversity
 }
-
 ###c_malta
 2646 = {	#MALTA
 	culture = punic
 	religion = nicene
 	holding = castle_holding
+	buildings = {
+		common_tradeport_01
+		cereal_fields_00
+	}
+	550.1.1 = { culture = african_romance }
 }
 
 ##d_corsica ###################################
+
 ###c_ajaccio
 2650 = {	#AJACCIO
 	culture = corsican
```

Comment:
It doesn't represent Paleo-Corsican but rather Medieval Corsican

Reply:
Then shouldn't it be in the Latin group instead of the Torrean Group? Torrean implies pre-Latin Corsican civilization to me.

Reply:
In my opinion yes,